File: mvc/src/main/java/com/example/mvc/PersonRepository.java
package com.example.mvc;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

/**
 * @author qiuchangdong
 * @Type PersonRepository
 * @Desc 。
 * @date 2021-03-10 14:45:47
 */
@Repository
public interface PersonRepository extends CrudRepository<Person, Integer> {
}

File: webflux/src/main/kotlin/com/example/webflux/PersonRepository.kt
package com.example.webflux

import org.springframework.data.repository.kotlin.CoroutineCrudRepository

/**
 * @author qiuchangdong
 * @Type PersonRepository
 * @Desc 。
 * @date 2021-03-10 10:18:28
 */
interface PersonRepository: CoroutineCrudRepository<Person, Int> {
}